Testing/VersionMinerTests/GitHubActionTests.cs

```diff
@@ -172,6 +172,29 @@ public async void Run_WhenInvoked_ShowsWelcomeMessageInCorrectOrder()
         actualExecutionOrder.Should().ContainInOrder(expectedOrder);
     }
 
+    [Fact]
+    public async void Run_WithTrimStartFromBranchSetToValue_TrimsBranchName()
```

Comment:
Search code base for the words **file type**.   They might be required to be **file format**.